### DISCUSSION TITLE
Compatibility of the dp potential in the latest deepmd-kit

### DISCUSSION THREAD

**Discussion by xn-huang** [deepmodeling/deepmd-kit#1436] [Q&A]
Hello. I met some problems in the installation and running of the latest deepmd-kit.
1. I used the same command [conda create -n deepmd deepmd-kit=*=*cpu libdeepmd=*=*cpu lammps-dp -c https://conda.deepmodeling.org] to install the latest deepmd-kit in two supercomputing clusters. One with the system of [OS: Linux "CentOS Linux 7 (Core)" 3.10.0-862.14.4.el7] can run lammps using dp potential normally, but the other with the system of [OS: Linux "Red Hat Enterprise Linux Server 7.3 (Maipo)" 3.10] can't. And the error is "ERROR: Unrecognized pair style 'deepmd' (src/force.cpp:278)". 
Is it due to the different system? How to fix it?

 2. In the supercomputing cluster where lammps deepmd package works normally, I trained a potential and it can be used in lammps. But when I used the potential trained by others using deepmd v2.0.3, error occurred as follows: Invalid argument: Tensor fitting_attr/dfparam:0, specified in either f or fetch_devices was not found in the Graph terminate called after throwing an instance of 'deepmd::tf_exception' what():  std::exception Aborted. 
 I am wondering why potentials are incompatible and how to fix the problem. 
Thanks

**Comment by njzjz** [comment]
You can execute `conda list` to check whether the same version was installed into two machines.

**Reply by xn-huang** [reply]
Thanks for your timely reply. 
The versions are all the same. We found that the compressed potential can't be used in different clusters, but the uncompressed one can.
But the other problem still exists. I installed the latest deepmd-kit in one supercomputing cluster. The deepmd package seems not to be installed. When I run lammps, the error occurs that  "ERROR: Unrecognized pair style 'deepmd' (src/force.cpp:278)". Do you have any suggestions to fix it? Thanks.

**Reply by njzjz** [reply]
The compressing bug is known and has been fixed by #1306.

For `unrecognized pair style`, you may check if you run correct LAMMPS? You can also [manually load deepmd pair style](https://docs.deepmodeling.org/projects/deepmd/en/v2.0.3/third-party/lammps-command.html#enable-deepmd-kit-plugin-plugin-mode).

**Reply by xn-huang** [reply]
Thanks for your timely reply.
I found that "unrecognized pair style" error is due to "Open of file libdeepmd_lmp.so failed: /BIGDATA2/caep_zqqiao_1/.conda/envs/dpmd_v2/bin/../lib/././libtensorflow_cc.so.2: symbol GOMP_parallel, version VERSION not defined in file libiomp5.so with link time reference". The reason is due to I didn't load intelcompiler before run lammps. After I loaded intelcompiler, lammps can work normally. 
Thanks again.
